README.rst

```diff
@@ -22,10 +22,14 @@ Usage
 
 ::
 
-    ❯ python main.py --help
+    ❯ pip install .
+    ❯ xapi-db-load --help
     Usage: main.py [OPTIONS]
 
     Options:
```

Comment:
This help is a bit out of date...

```suggestion
    Options:
  --backend [clickhouse|mongo|citus|ralph_clickhouse|ralph_mongo]
                                  Which backend to run against  [required]
  --num_batches INTEGER           Number of batches to run, num_batches *
                                  batch_size is the total rows
  --batch_size INTEGER            Number of rows to insert per batch,
                                  num_batches * batch_size is the total rows
  --drop_tables_first BOOLEAN     If True, the target tables will be dropped
                                  if they already exist
  --distributions_only BOOLEAN    Just run distribution queries and exit
  --db_host TEXT                  Database host name
  --db_port TEXT                  Database port
  --db_name TEXT                  Database name
  --db_username TEXT              Database username
  --db_password TEXT              Password for the database so it's not stored
                                  on disk
  --lrs_url TEXT                  URL to the LRS, if used
  --lrs_username TEXT             LRS username
  --lrs_password TEXT             Password for the LRS so it's not stored on
                                  disk
  --help                          Show this message and exit.
```

Reply:
@bmtcril Hmm.. Since it's probably going to continue to be out of date, I think it's better to remove the USAGE from the README. What do you think?

Reply:
Makes sense, I've added https://github.com/openedx/xapi-db-load/issues/3 to cover writing some more formal docs and a slew of other housekeeping items.